index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Identities
```

Comment:
```suggestion
      Digital Identity API
```

Maybe more consistent with other spec names? That is, it is an API specifically, rather than an overview of digital identities broadly speaking?

Reply:
We've generally moved away from having "API" in the names of specs (e.g., "Credential Management", "Permissions", "Fetch", etc.) to cover both the generalized concepts and the API.

Reply:
Ha, didn't realize that was the case. I'll update a few more of my specs that I maintain to follow that then.

Resolving.